unison-cli/src/Unison/CommandLine/InputPatterns.hs

```diff
@@ -2270,13 +2270,13 @@ deprecatedViewRootReflog =
 branchReflog :: InputPattern
 branchReflog =
   InputPattern
-    "branch.reflog"
-    ["reflog.branch", "reflog"]
+    "reflog"
+    ["reflog.branch", "branch.reflog"]
     I.Visible
     []
     ( P.lines
-        [ "`branch.reflog` lists all the changes that have affected the current branch.",
-          "`branch.reflog /mybranch` lists all the changes that have affected /mybranch."
+        [ "`reflog` lists all the changes that have affected the current branch.",
+          "`reflog /mybranch` lists all the changes that have affected /mybranch."
```

Comment:
No need to change this now, but just a heads up that I think https://github.com/unisonweb/unison/pull/5180 cleared the path for using `makeExample` in here too.